dbms/src/TableFunctions/TableFunctionRandom.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <TableFunctions/ITableFunction.h>
+
+namespace DB
+{
+/* random(structure, limit) - creates a temporary storage filling columns with random data
```

Comment:
limit can be optional, so we just write LIMIT in SELECT or wait for something to happen.

Reply:
done